modules/displayDateAndTime.js

```diff
@@ -0,0 +1,7 @@
+import { dateDisplay } from './dom.js';
+import { DateTime } from './luxon.min.js';
+
+export default function displayCurrentDateAndTime() {
+  const dt = DateTime.now();
+  dateDisplay.innerHTML = dt.toLocaleString(DateTime.DATETIME_FULL_WITH_SECONDS);
+}
```

Comment:
- [ ] Same here, please use the `ES6` arrow function syntax instead of regular syntax.